src/jpegParser.js

```diff
@@ -74,15 +69,22 @@
       previousByte = currentByte;
       offset += 1;
     }
-    return offset + 1;
+    return offset - 1;
```

Comment:
Segment start at the 0xff marker. But we already read it on counted it. So we need to track back to include it.

This is not Exif, it is JPEG.